pilz_status_indicator_rqt/src/pilz_status_indicator_rqt/status_indicator.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+# Copyright (c) 2020 Pilz GmbH & Co. KG
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+
+import rospy
+from pilz_status_indicator_rqt.status_indicator_widget import \
+    PilzStatusIndicatorWidget
+from prbt_hardware_support.msg import OperationModes
+from qt_gui.plugin import Plugin
+from std_msgs.msg import Bool, Float64
+
+TOPIC_OPERATION_MODE = "/prbt/operation_mode"
+TOPIC_DIAGNOSTICS_PRBT = "/prbt/diagnostics/state_prbt"
```

Comment:
These topic names can be misleading since they have nothing todo with http://wiki.ros.org/diagnostic_msgs
I suggest on of the following options:
1. Rename the topic
2. Subscribe to diagnostics of type http://docs.ros.org/api/diagnostic_msgs/html/msg/DiagnosticArray.html and change the relevant code parts 

Note: Run https://github.com/ros/diagnostics/blob/indigo-devel/diagnostic_updater/src/example.py to see an example of the messages published on /diagnostics and maybe use it as a base for writing a mock to test the gui...